packages/components/atoms/f-form-field/src/assets/img/icons/check--grey.svg

```diff
@@ -0,0 +1,3 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 8">
```

Comment:
Not necessarily for this PR, but as an improvement – would there be a way to add the file to f-icons and pull it in from the `node_modules` folder? All the icons in f-icons should be available as part of the dist folder in that package.

Reply:
I wasn't sure about adding whole f-icon package for 2 small icons. And for most of icons we can change colour via css and in this case I added 2 same tick icons with different fill colour because it is used as a css background image.

Reply:
The whole icon-set shouldn't get bundled – only the ones you actually bundle inline. 

You might be able to use CSS filters to change the colour of solid icons like this too – imagine it would be ok for grey and white like this.

Reply:
Nice, I will create a separate ticket for this :+1: